ansible/roles/microshift-start/defaults/main.yml

```diff
@@ -1,7 +1,6 @@
----
 # microshift-start default vars
 
-remote_kubeconfig_path: /var/lib/microshift/resources/kubeadmin/kubeconfig
+remote_kubeconfig_path: /var/lib/microshift/live/resources/kubeadmin/kubeconfig
```

Comment:
We should think about whether it would be helpful to have symlinks to the old locations for these files, too. That will cut down on some confusion for someone seeing old docs.

Reply:
Having symlinks allows for the old and new structure to co-exist. I think may want to prefer a clear-cut between 4.13 and 4.14 versions to "force" users to migrate to the new structure which is going to be the "production" one.

Reply:
If we want to preserve the backward compatibility, I would not introduce the "live" sub-directory then, keeping the backup in `/var/lib/microshift-backup` parallel dir.